openai-client/generator/config/model_oneof.mustache

```diff
@@ -24,10 +24,11 @@ object {{classname}}Serializer : KSerializer<{{classname}}> {
     }
 
     override fun deserialize(decoder: Decoder): {{classname}} {
+      val jsonSerde = Json { ignoreUnknownKeys = true }
```

Comment:
Hmm. Should we go back to a global instance instead? It's meant for OpenAI specifically anyway here, and recreating the `Json` is quite expensive (I think it initialises some reflection stuff).

Can also take care of this in a subsequent PR, there is probably more cases where we want to change hardcoded instances of `Json`.

Reply:
Yes, we need to go back to a global instance, which is probably better done in another PR. ChatWithFunctions also uses a new instance each time for deserializing the LLM function call, and sometimes, it fails because it's not permissive enough. That one also needs config for `ignoreUnknownKeys`.

We also noticed the serializers are using the `type` property defined twice in objects with a property with the same name. We may have to change the `classDiscriminator` globally so it is not called `type` in the Json object config for all calls.

Reply:
merging this now, let's do this in another PR